lib/engine/game/g_18_dixie/step/icgscl_merge_consent.rb

```diff
@@ -0,0 +1,144 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/base'
+
+module Engine
+  module Game
+    module G18Dixie
+      module Step
+        class MergeConsent < Engine::Step::Base
+          def round_state
```

Comment:
After perusing your changes and the [18Dixie Player Aid on BGG](https://boardgamegeek.com/filepage/132102/18dixie-player-aid), this is what I've come up with for the merger stuff.

I think if you didn't want this on the round state, you could have something like this on the game class:

```
attr_accessor :merger

@merger = {
  consent: {
    merging_corp: nil,
    primary_corp: nil,
    secondary_corp: nil,
    pending_corps: [],
    subsidy: nil,
  },
  presidency: {
    cash_crisis: {
      corp: nil,
      cash_crisis_player: nil,
    },
    exchange: {
      corps: [],
      merging_corp: nil,
      subsidy: nil,
    },
  },
}
```

All the steps would be able to access what they needed with `@game.merger[:presidency]` or whatever. Don't know if that's really any less clunky than the round state approach 🤷